slo-reporter/overlays/cnv-prod/secrets.enc.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 items:
 -   apiVersion: v1
     data:
-        token: ENC[AES256_GCM,data:jkr65NIcWScSOi20cKSSkCRTPyXpkUDPgOL+5X4ZE14ZYs9TeHxPAf1zUiNcpSIEj6dq9d/gfgDgK/bCxVLgTX/5rnvyEioyQuvJMe0HRST+SvzJ+0+fuUam37kA3KZs6Dhv1jbYKfY9AIM5iW7L+Hpd8V7o614P0hxuOIpJs9QmYqdTkuH5hliau2/c2Tb4tfigceLcx6ipEQRaYs5R4RkSMCI64AOYw/DoAtIz7rPm4801Dj2IhbwFQqOwHuXfY8aGCo8rKkWnnRecUu9w0BtmaVGK938Z9IYhZrQq6S5v1wdu2jwUMeLjggP7in+nAD/eUwgaBFzoJIa7jBKaCAWTxXy0JBNYp4INFQk5D/ks/Y9/ZdUlfx6Z4MOuze0luQj3Du7ccIVKmv5oYwC9QrRFe52FIrPlP7/anMPKhIjyiDS1wU8SoZPs+dPgRukHmDSXnOQVjk7sQnsedElcjRVPsd/Y1Fn5SKAVZ+iICznTvmy4AziGbl2za0CtB4/vI6C5BdkTuUKbYIsiixe62EqOLPjMleSu59eXlMjlm/sLGKB2mjsNq4UzTgb+pXdObyLtXS/ikxQQ3FhGg0z3EdKbMfSwkg/+V++aSNsDIw++sRGA0C/khfk+VloZA8An/8kEYMDEsRHd5oRaebAfc45Cvon0w2/eRXX0Q+RJ7tqYzNH8Amk3cowHENgogAnZ7xd+kbGCA5UCrjk1aMJu2X575Ip8FzQEkQ3DNK7F+Q8rcloBRG0MLaVDknjANY+a+YYe/YQHEfwjY71C161YhPM913TNYOw1x779uFEwSdOwmS4moXu11dUkf2oag7vVE3jW/DX8kQOoez6VuQLnH6Ofg8dH25g5KDTOkDkUlUGAQ8z0LXYsZe7+Y+bwSdZWhxCN3AaUs0nQYpmRIgkSKpcMOcLSdvSbZWJ0HkgyVYmwLRRqAmuWifRkw8dd4qcDmoCrmSc6oS1ABHvguhUk6TyDnr+xP5VR/0qpwUhSxTtdYOI4Eyxr4M5ozSh411YK443XZHs8ehWBGdmNDWtj2kbzeL2FzXpFbJQcl28/0g777/FVulS5nAkFXhypCspije0+62rBW2JCACqO4Z4lFAXK8LdF2YypX3/vQpnZiBvbdZszcokcdWY91J0++iIKm3IzgLoOBfW6bIsJSIHhigBLWdkHuhchOrqBtJZKlIzk+e6OJJxGTJBrNRNcyvcQqtiwXWFXZNyJxI7VxWNN3njbldOt3ZtO3r9dJd5Lbyy/LmL5c7cl0Id5N/rAyeY/H9s8UpZ001x9mGGxtLA5SZiMKCaDVfCk3/2gWD2UgIZdvxk8CAt+TunKNPGM2PBZwlwsXJkJsCCMX4RRR43rSRHlZ6K6DIWvNyX7BL5ZXmsQF+6H0AuRckjfn8mNCOAea51NV/vTMoDxPMLAM824f4xuZAsmewP4mcqx1PdkKYFcT7FQF+tEeeQFK59txFPaOp4IjyG3GSf8kfusnTssgEMRvOYIxzoFiGtzJXXA/U4AymNJo5ghhtITdP7+7Er7J7WAcTI7ZYUgmjVeigh+4CPDXlvTm3fmbtSeEcgVVZCOc9bS+VL9BumdJMZ3oTL7b6NfF10Ybislnp9X5MYuHkc+ZyqVwGG2ZYtWwpjXGo4=,iv:VxUDHZmxkBCd4WsMJvw2onBbtPOm1mFZRRAIONcJPT0=,tag:no2lBRVMnGLDU/JDSnL8uw==,type:str]
```

Comment:
what is this token referencing too?

Reply:
See conversation here: https://github.com/thoth-station/thoth-application/pull/879#discussion_r573189699

Related-To: https://github.com/thoth-station/thoth-application/issues/920